.github/workflows/deploy-gitea-runners.yaml

```
@@ -0,0 +1,126 @@
+          if [ "${{ github.ref }}" == "refs/heads/main" ]; then
+            echo "tag=${{ needs.get-short-sha.outputs.short-sha }}" >> $GITHUB_OUTPUT
+          else
+            sanitized_branch_name=$(echo "${{ github.ref_name }}" | tr -d '()' | tr '/' '-')
+            echo "tag=${sanitized_branch_name}-${{ needs.get-short-sha.outputs.short-sha }}" >> $GITHUB_OUTPUT
+          fi
+
+  test-matrix:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        orgenvironment: ${{ fromJSON(needs.construct-org-environments-array.outputs.orgenvironmentsjson) }}
```

actionlint (1.7.7)

property "construct-org-environments-array" is not defined in object type {}

(expression)